db_stress_tool/db_stress_gflags.cc

```diff
@@ -346,6 +346,10 @@ DEFINE_int32(
 DEFINE_bool(use_clock_cache, false,
             "Replace default LRU block cache with clock cache.");
 
+DEFINE_bool(
+    use_fast_lru_cache, false,
```

Comment:
It might be better to use an enum to specify the desired cache type, rather than separate boolean flags for fast_lru_cache and clock_cache. I know currently we don't test the clock cache because its broken, but when we fix it it would be hard to enforce the mutual exclusion in db_crashtest.py.

Reply:
Fixed.